socorro/unittest/external/boto/test_connection_context.py

```diff
@@ -53,40 +48,31 @@ class ConditionallyABadDeal(Exception):
 thing_as_str = json.dumps(a_thing)
 
 
-class ConnectionContextTestCase(socorro.unittest.testbase.TestCase):
-    def setup_mocked_s3_storage(
-        self,
-        executor=TransactionExecutor,
-        executor_for_gets=TransactionExecutor,
-        storage_class='BotoS3CrashStorage',
-        host='',
-        port=0,
-        resource_class=S3ConnectionContext,
-        **extra
-    ):
-        config = DotDict({
-            'resource_class': resource_class,
-            'logger': mock.Mock(),
-            'host': host,
-            'port': port,
-            'access_key': 'this is the access key',
-            'secret_access_key': 'secrets',
-            'bucket_name': 'silliness',
-            'keybuilder_class': KeyBuilderBase,
-            'prefix': 'dev',
-            'calling_format': mock.Mock()
-        })
```

Comment:
Using `get_config` lets us not have to specify values for all the defaults. This makes our tests much better.